podman.spec.rpkg

```diff
@@ -77,12 +77,15 @@ BuildRequires: ostree-devel
 BuildRequires: systemd
 BuildRequires: systemd-devel
 Requires: conmon >= 2:2.0.30-2
-Requires: containers-common >= 4:1-30
-Requires: containernetworking-plugins >= 1.0.0-15.1
+# containers-common pulled from podman-next copr for f34,
+# from the distro repos for f35+
+%if 0%{?fedora} <= 35
+Requires: containers-common >= 4:1-39
```

Comment:
The last bodhi I see for fc35 is -36 and it's not even in repos yet; for fc34 it's -21. Won't this make the package uninstallable?

Reply:
@edsantiago RE: f34, I sent a containers-common to the podman-next copr. https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/build/3289552/ . Also, for f35, there's -39 at https://bodhi.fedoraproject.org/updates/FEDORA-2022-0afb41b09c which got struck down. But it should still work ok with non-moby users.